options.go

```diff
@@ -90,6 +90,7 @@ func (rc *InstanceOptions) init(opts ...InstanceOption) error {
 		log.Infof("[Config Center] Config center doesn't start")
 		log.Debugf("config center doesn't start because %s", err)
 	} else {
+		compatInstanceOptions(rcCompat, rc)
```

Comment:
Is this the reverse operation of `compatRootConfig(c *InstanceOptions) *config.RootConfig`? This is to copy configurations in RootConfig into InstanceOptions while `compatRootConfig(c *InstanceOptions) *config.RootConfig` does the opposite.